src/UniqueCharacters.java

```diff
@@ -1,10 +1,19 @@
+import java.util.Scanner;
+
 public class UniqueCharacters {
     public static void main(String[] args) {
-        System.out.println(uniqueCharacters("caio"));
+        Scanner input = new Scanner(System.in);
+        System.out.println("Welcome to UniqueCharacters!!\nEnter a string!!");
+        String userInput = input.next();
+        if (uniqueCharacters(userInput)) {
+            System.out.println("It's " + uniqueCharacters(userInput) + ", this string has unique characters.");
```

Comment:
to avoid call multiple times uniqueCharacters method, please get the return and use a local variable

Reply:
Done.